### PR TITLE
[release/1.0.1] Redist old build of System.Private.Uri

### PR DESCRIPTION
**Pull request by ericstj**
System.Private.Uri is part of the shared library and cannot be updated.

Port of https://github.com/dotnet/corefx/commit/5da5eaa790a813a90d76b308f3c40e7b4a5a46a7

/cc @weshaggard @MattWhilden

### CODE REVIEW

**Comment by ericstj** [comment]
@dotnet-bot test this please.

**Comment by ericstj** [comment]
@dotnet-bot test this please

**Comment by ericstj** [comment]
@mmitche anything I need to do to get CI working here?

**Comment by mmitche** [comment]
@ericstj Looks like it's working

File: src/System.Private.Uri/src/redist/project.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "dependencies": {
+    "Microsoft.NETCore.Targets": "1.0.0",
```
Comment (weshaggard) [inline src/System.Private.Uri/src/redist/project.json]: Do we actually need the line-up here?

**Comment by weshaggard** [comment]
LGTM

**Comment by MattWhilden** [comment]
LGTM.

File: src/System.Private.Uri/src/redist/project.json (hunk 1 above)
Comment (ericstj) [inline src/System.Private.Uri/src/redist/project.json]: That's pulling in the RID graph.  It wasn't automatically referenced for 1.0.0.

**Comment by ericstj** [comment]
OSX failing with https://github.com/dotnet/corefx/issues/9066.
@dotnet-bot test Innerloop OSX Debug Build and Test

**Comment by mmitche** [comment]
@ericstj Maybe that machine is horked

@dotnet-bot test Innerloop OSX Debug Build and Test

**Comment by ericstj** [comment]
Closing this because we're actually going to reuse the 1.0.0 branch.

**Pull request closed without merging**